src/classes/BDI_MigrationMappingHelper.cls

```diff
@@ -130,10 +148,12 @@ public class BDI_MigrationMappingHelper {
                     Target_Object_Mapping__c,
                     Target_Object_Mapping__r.Object_API_Name__c,
                     Target_Object_Mapping__r.DeveloperName,
+                    Target_OBject_Mapping__r.Legacy_Data_Import_Object_Name__c,
                     Target_Object_Mapping__r.MasterLabel,
                     Is_Deleted__c
             FROM Data_Import_Field_Mapping__mdt
-            WHERE Data_Import_Field_Mapping_Set__r.DeveloperName = :developerName];
+            WHERE Data_Import_Field_Mapping_Set__r.DeveloperName = :developerName
+            AND Is_Deleted__c = false];
```

Comment:
Super minor comment, but can you indent this "AND" so that it's one tab further to the right?

Reply:
All set!